File: app/src/main/java/com/mappls/sdk/demo/java/activity/MapFragmentActivity.java
package com.mappls.sdk.demo.java.activity;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import com.mappls.sdk.demo.databinding.ActivityMapFragmentBinding;
import com.mappls.sdk.demo.java.fragments.MapFragment;

public class MapFragmentActivity extends AppCompatActivity {

    private ActivityMapFragmentBinding mBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBinding = ActivityMapFragmentBinding.inflate(getLayoutInflater());

        setContentView(mBinding.getRoot());
        getSupportFragmentManager().beginTransaction()
                .add(mBinding.containerId.getId(), new MapFragment(), MapFragment.class.getSimpleName())
                .commit();
    }
}

File: app/src/main/java/com/mappls/sdk/demo/kotlin/activity/MapFragmentActivity.kt
package com.mappls.sdk.demo.kotlin.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.mappls.sdk.demo.R
import com.mappls.sdk.demo.kotlin.fragments.MapFragment

class MapFragmentActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_map_fragment)

        supportFragmentManager.beginTransaction().add(R.id.container_id, MapFragment(), MapFragment::class.java.simpleName).commit()
    }
}